OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -57,6 +57,9 @@ public static void FlyToward(Actor self, Aircraft plane, int desiredFacing, WDis
 
 		public override Activity Tick(Actor self)
 		{
+			// Refuse to take off if it would land immediately again.
+			if (plane.ForceLanding) return null;
```

Comment:
Doesn't [OpenRA.Game/Activities/Activity.cs#L213](https://github.com/OpenRA/OpenRA/blob/fc0495ac2793172691227405c25b32c3db89db3a/OpenRA.Game/Activities/Activity.cs#L213) do the same?